### PR TITLE
Site Settings: Paging toggle and close after fix

### PR DESCRIPTION
**Pull request by rishabh7m**
fix for #3642

### CODE REVIEW

File: WordPress/src/main/java/org/wordpress/android/ui/prefs/SiteSettingsInterface.java [hunk 1]
```diff
@@ -418,6 +422,10 @@ public int getPagingCount() {
     public @NonNull String getPagingDescription() {
         if (mActivity == null) return "";
 
+        if (!getShouldPageComments()) {
+            return mActivity.getString(R.string.none);
```
Comment (maxme) [inline WordPress/src/main/java/org/wordpress/android/ui/prefs/SiteSettingsInterface.java]: Should we display "disabled" instead of "none"? cc @tonyr59h

Comment (rishabh7m) [inline WordPress/src/main/java/org/wordpress/android/ui/prefs/SiteSettingsInterface.java]: Yup, I also think that would be more appropriate

**Comment by rishabh7m** [comment]
HI @maxme , should I change "none" to "disable"?

File: WordPress/src/main/java/org/wordpress/android/ui/prefs/SiteSettingsInterface.java (hunk 1 above)
Comment (tonyr59h) [inline WordPress/src/main/java/org/wordpress/android/ui/prefs/SiteSettingsInterface.java]: I think 'disabled' makes more sense here and chatted with @mattmiklic about it. He agrees, let's switch it to 'Disabled'.

**Comment by tonyr59h** [comment]
@rishabh7m go ahead and change the string to 'Disable' :+1:

**Comment by maxme** [comment]
Looks good :shipit:

**Pull request merged**